misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java

```diff
@@ -270,7 +274,9 @@ private JsonObject visibleEP(Node node, TypeSymbol typeSymbol, boolean isRemoteA
             symbolMetaInfo.addProperty("isCaller", "Caller".equals(typeSymbol.getName().orElse(null)));
```

Comment:
Do not use `orElse(null)` this bypasses the advantage of optional, hence use `orElse("")`